src/test/java/org/myrobotlab/service/SolrTest.java

```diff
@@ -1,11 +1,12 @@
 package org.myrobotlab.service;
 
-import static org.bytedeco.opencv.global.opencv_imgcodecs.cvLoadImage;
+import static org.bytedeco.opencv.helper.opencv_imgcodecs.cvLoadImage;

```

Comment:
Why is this import changing as a result of the PR ?  we didn't switch opencv versions ...

Reply:
hmm.. setting back